client/my-sites/themes/thanks-modal.jsx

```diff
@@ -75,7 +76,7 @@ class ThanksModal extends Component {
 
 	visitSite = () => {
 		this.trackClick( 'visit site' );
-		page( this.props.visitSiteUrl );
```

Comment:
😮 Where did we even get this?

Reply:
I don't know! `visitSite` is never called, and this is the only instance in the **entire** codebase of `visitSiteUrl`! It's not even passed through props anywhere.